src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -785,6 +806,8 @@ public function send(string $method, string $url, array $options = [])
             $url = ltrim(rtrim($this->baseUrl, '/').'/'.ltrim($url, '/'), '/');
         }
 
+        $url = $this->expandUrlParameters($url);
```

Comment:
would it make sense to validate the params, to make sure it match the url template?